api/v1alpha1/backend_types.go

```diff
@@ -117,7 +117,7 @@ type UnixSocket struct {
 	// Path defines the unix domain socket path of the backend endpoint.
 	// The path length must not exceed 108 characters.
 	//
-	// +kubebuilder:validation:MaxLength=108
+	// +kubebuilder:validation:XValidation:rule="size(self) <= 108",message="unix domain socket path must not exceed 108 characters"
```

Comment:
Prev implement cause unexpected error:
```console
The Backend "backend-unix" is invalid:
* spec.endpoints[0].unix.path: Too long: may not be more than 108 bytes
* <nil>: Invalid value: "null": some validation rules were not checked because the object was invalid; correct the existing errors to complete validation
```